shared/src/test/scala/uk/gov/homeoffice/drt/arrivals/ArrivalSpec.scala

```diff
@@ -0,0 +1,120 @@
+      val arrival = arrivalBase.copy(TotalPax = SortedSet(
+        liveFeedTotalPaxSource,
+        portForecastFeedTotalPaxSource,
+        apiFeedWithSplitsTotalPaxSource,
+        apiFeedWithHistoricalSplitsTotalPaxSource,
+        apiFeedWithOutSplitsTotalPaxSource,
+        aclFeedTotalPaxSource))
+      arrival.bestPcpPaxEstimate mustEqual liveFeedTotalPaxSource
+    }
+
+    "When totalPax" +
+      " does not contain Live feed source" +
+      " bestPcpPaxEstimate gives apiFeed With ApiSplitsWithHistoricalEGateAndFTPercentages" in {
+      val arrival = arrivalBase.copy(TotalPax = SortedSet(
+        portForecastFeedTotalPaxSource,
+        apiFeedWithSplitsTotalPaxSource,
+        apiFeedWithHistoricalSplitsTotalPaxSource,
+        apiFeedWithOutSplitsTotalPaxSource,
+        aclFeedTotalPaxSource))
+      arrival.bestPcpPaxEstimate mustEqual apiFeedWithSplitsTotalPaxSource
+    }
+
+    "When totalPax " +
+      " does not contain Live feed source and ApiFeed Source with ApiSplitsWithHistoricalEGateAndFTPercentages" +
+      " bestPcpPaxEstimate gives port forecast feed" in {
+      val arrival = arrivalBase.copy(TotalPax = SortedSet(
+        portForecastFeedTotalPaxSource,
+        apiFeedWithHistoricalSplitsTotalPaxSource,
+        apiFeedWithOutSplitsTotalPaxSource,
+        aclFeedTotalPaxSource))
+      arrival.bestPcpPaxEstimate mustEqual portForecastFeedTotalPaxSource
+    }
+
+    "When totalPax" +
+      " does not contain Live feed source and ApiFeed Source with ApiSplitsWithHistoricalEGateAndFTPercentages" +
+      " bestPcpPaxEstimate gives port forecast feed" in {
+      val arrival = arrivalBase.copy(TotalPax = SortedSet(
+        portForecastFeedTotalPaxSource,
+        apiFeedWithHistoricalSplitsTotalPaxSource,
+        apiFeedWithOutSplitsTotalPaxSource,
+        aclFeedTotalPaxSource))
+      arrival.bestPcpPaxEstimate mustEqual portForecastFeedTotalPaxSource
+    }
+
+    "When totalPax" +
+      " does not contain Live feed source , ApiFeed Source with ApiSplitsWithHistoricalEGateAndFTPercentages and port forecast feed source" +
+      " bestPcpPaxEstimate gives api with historic feed" in {
+      val arrival = arrivalBase.copy(TotalPax = SortedSet(
+        apiFeedWithHistoricalSplitsTotalPaxSource,
+        apiFeedWithOutSplitsTotalPaxSource,
+        aclFeedTotalPaxSource))
+      arrival.bestPcpPaxEstimate mustEqual apiFeedWithHistoricalSplitsTotalPaxSource
+    }
+
+    "When totalPax" +
+      " does not contain Live feed source , ApiFeed Source with ApiSplitsWithHistoricalEGateAndFTPercentages , ApiFeed Source with Historical and port forecast feed source" +
+      " bestPcpPaxEstimate gives apiFeed without splitsSource feed" in {
+      val arrival = arrivalBase.copy(TotalPax = SortedSet(
+        apiFeedWithOutSplitsTotalPaxSource,
+        aclFeedTotalPaxSource))
+      arrival.bestPcpPaxEstimate mustEqual apiFeedWithOutSplitsTotalPaxSource
+    }
+
```

Comment:
I guess we should add a test for the case when there's only ACL for completeness